my-app/src/cuisine.js

```diff
@@ -74,7 +74,7 @@ function Cuisine() {
                 const imageDiv = document.createElement("div");
                 imageDiv.classList.add("restaurantImage")
                 const restaurantImg = document.createElement("img");
-                restaurantImg.src = require('./Images/' + searchParams.get('nationality') + '.jpg');
+                restaurantImg.src = results[i].photos[0].getUrl()

```

Comment:
Sorry. I think I missed this before but because the scope of the for loop is pretty large, can you rename the variable i to something like resultIdx?